packages/fxa-content-server/server/.eslintrc

```diff
@@ -1,8 +1,8 @@
-plugins:
-  - fxa
-extends: ../.eslintrc
-
-rules:
-  id-blacklist: 0
-  no-var: 2
-  strict: 0
+{
+  "extends": "../.eslintrc",
+  "rules": {
+    "id-blacklist": 0,
+    "no-var": 2,
+    "strict": 0
+  }
```

Comment:
Is there a reason the .eslintrc here is pretty different from the others? It's fine if we need to special case content-server, just want to check.

Reply:
These were a couple of rules with huge breakages in the server code. I felt it was too invasive of a change for this pr. Though, I don't know that it would even be worth addressing in the future